build.rs

```diff
@@ -5,9 +5,9 @@ fn main() {
         println!("cargo:rustc-link-lib=framework=AppKit");
     }
 
-    // if std::env::var("TARGET").unwrap().contains("-ios") {
-    //     println!("cargo:rustc-link-lib=framework=UIKit");
-    // } else {
-    //     println!("cargo:rustc-link-lib=framework=AppKit");
-    // }
+    #[cfg(all(feature = "gtk3", feature = "xdg-portal"))]
+    compile_error!("You can't enable both GTK3 & XDG Portal features at once");
+
+    #[cfg(not(any(feature = "gtk3", feature = "xdg-portal")))]
+    compile_error!("You need to choose at least one backend: `gtk3` or `xdg-portal` features");
```

Comment:
I like this better than silently choosing a fallback.

Reply:
Yeah, it's better to just be explicit, glad that we agree.

Reply:
Yeah, it's better to just be explicit, glad that we agree.